soh/soh/Enhancements/randomizer/randomizer_entrance_tracker.cpp

```diff
@@ -846,7 +850,11 @@ void DrawEntranceTracker(bool& open) {
                     uint32_t color = isDiscovered ? IM_COL32_WHITE : COLOR_GRAY;
 
                     // Handle highlighting and auto scroll
-                    if (LinkIsInArea(original) != -1) {
+                    if (original->index == lastEntranceIndex ||
+                        (override->reverseIndex == lastEntranceIndex && OTRGlobals::Instance->gRandomizer->GetRandoSettingValue(RSK_DECOUPLED_ENTRANCES) == RO_GENERIC_OFF) &&
+                            CVarGetInteger("gEntranceTrackerHighlightPrevious", 0)) {
```

Comment:
Just a nit-pick, but could you wrap the two OR-ed statements here in parentheses
```suggestion
                    if ((original->index == lastEntranceIndex ||
                        (override->reverseIndex == lastEntranceIndex && OTRGlobals::Instance->gRandomizer->GetRandoSettingValue(RSK_DECOUPLED_ENTRANCES) == RO_GENERIC_OFF)) &&
```

Reply:
I don't know why this comment had three different lines of code represented between the two changes, but I implemented it in my update.

Reply:
I messed up the comment/suggestion, and didn't want to redo it so I just removed the "suggestion". Thank you 🙂